### PR TITLE
[zh] Fix Markdown format

### PR DESCRIPTION
**Pull request by Sea-n**
- Change `` `hello,` world `` to `` `hello`, world ``
- Use correct `codenew` statement

Ref: #32897

### CODE REVIEW

**Comment by Sea-n** [comment]
/cc @tengqm

Could you please help with this PR? :bowing_man:

File: content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ If you are running an HA cluster, this command needs to be executed on all the c
 {{< /warning >}}
 
 <!-- 
-` certs renew` uses the existing certificates as the authoritative source for attributes (Common Name, Organization, SAN, etc.) instead of the kubeadm-config ConfigMap. It is strongly recommended to keep them both in sync.
+`certs renew` uses the existing certificates as the authoritative source for attributes (Common Name, Organization, SAN, etc.) instead of the kubeadm-config ConfigMap. It is strongly recommended to keep them both in sync.
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md:267]: /hold

谢谢贡献。根据[指南](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87)约定，我们会拒绝对页面的**部分**修改。部分修改针对的是对应页面的英文上游还有其他改动需要同步的情况。
一旦我们合并了此 PR，将来就无法根据文件的最后更改时间来判断是否有其他上游变更尚未同步。请理解。

请在 **main** 分支上执行以下命令查看上游尚未同步的更改：

```shell
./scripts/lsync.sh content/zh/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md
```

**Comment by tengqm** [comment]
Other than the one which is now out of sync. The rest of the changes are all okay to me.

**Comment by Sea-n** [comment]
Thanks for letting me know the `kubeadm-certs.md` file could cause issues for future translating!
The change made for that file is inside the HTML comment (for comparing original text), so it's ignorable and I removed that change.

Please take a look at the new change, thanks for your time!

**Comment by tengqm** [comment]
These are all good.
/lgtm
/approve

**Comment by Sea-n** [comment]
/unhold

Thanks @tengqm for reviewing this, have a nice weekend!

**Pull request merged**